cinder_service_check.py

```diff
@@ -53,6 +53,12 @@ def check(auth_ref, args):
 
     services = r.json()['services']
 
+    # We need to match against a host of X and X@lvm (or whatever backend)
+    if args.host:
+        backend = ''.join((args.host, '@'))
```

Comment:
I don't think it's that clear cut. I doubt it makes any real difference performance wise which makes it a discussion on style and I like the consistency of always doing it that way.